### PR TITLE
Basic info in README

### PR DESCRIPTION
**Pull request by swertz**
Add basic install instruction in the README. Anything else?

@blinkseb: I'm not sure about the minimal version & all requirements. Missing something?

### CODE REVIEW

**Comment by blinkseb** [comment]
Very nice thanks! Requirement seems ok. I'd remove `gcc` and state something like `a C++11 compatible compiler (tested with g++ 4.9 and 5.3)`

I'd also talk about the `-DCMAKE_INSTALL_PREFIX` option if you want to install `MoMEMta` elsewhere than the system directories

Something silly also, but I'd also add somewhere it's developed on `Linux` and was not tested on `OSX` nor `Windows`.

I'll make some comments on the text itself, but in any case, many thanks!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+
+### Prerequisites
+
+MoMEMta depends on the following libraries and tools:
+   * LHAPDF (>=6)
+   * CMake (>=2.8.8)
+   * Boost
+   * ROOT
+   * GCC (>=4.8)
+   * Lua (will be auto-installed if not available)
```
Comment (blinkseb) [inline README.md]: It's not really installed, we ship our own version with momemta but we don't install anything

File: README.md [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+
+If you have admin rights on your system, you can make MoMEMta (public headers and library) available system-wide using:
+```
+# make install
+```
+
+### Build options
+
+The following options are available when configuring the build (when running `cmake ..`):
+   * `-DPROFILING=ON`: Generate debugging symbols
```
Comment (blinkseb) [inline README.md]: You also need to have `gperftools` installed somewhere if you turn-on `PROFILING` (https://github.com/gperftools/gperftools)

**Comment by swertz** [comment]
Updated!

**Pull request merged**